### ISSUE TITLE
opentpomap tile server with english labels

### ISSUE THREAD

**Issue by comoglu** [der-stefan/OpenTopoMap#193]
Hi,
Do we have a public opentopomap tile server with english labels for all countries? 
thanks in advance

**Comment by der-stefan** [comment]
No, we just have local names worldwide. I am German and my own map is quite useless for me e.g. in Asia.
Some day - perhaps - if we are fortune - we will provide transparent overlays showing custom labels. But don't expect it in the next 10 years, if no diligent coding-Jesus appears and joins the OTM team. :roll_eyes:

**Comment by der-stefan** [comment]
This issue will be solved with vector maps - see #186.
